### PR TITLE
test: add docker helper test

### PR DESCRIPTION
**Pull request by bwanglzu**

### CODE REVIEW

File: codecov.yml [hunk 1]
```diff
@@ -1,6 +1,3 @@
-codecov:
```
Comment (JoanFM) [inline codecov.yml:1]: why this? should we isolate this in another PR?

Comment (bwanglzu) [reply codecov.yml:1]: ok, I was trying to experiment what we discussed in triage codecov channel, I'll revert this one.

Comment (JoanFM) [reply codecov.yml:1]: ah well, if it is for investigating is good, does it make any improvement?

Comment (bwanglzu) [reply codecov.yml:1]: @JoanFM I didn't see, because python 3.9 ci failed and... coverage report not being uploaded

**Comment by bwanglzu** [comment]
no..i messed up my pr, try to merge latest CI (without python 3.9 required)

**Pull request closed without merging**